aerospike_helpers/cdt_ctx.py

```diff
@@ -77,13 +77,41 @@
     print(result)
     # {'users': 4}
 
+    # Example 3: index_cdt_create() with info command
+    policy = {}
+
+    bs_b4_cdt = client.get_cdtctx_base64({'ctx':ctx_list_index})
+    
+    r = []
+    r.append("sindex-create:ns=test;set=demo;indexname=test_string_list_cdt_index")
+    # use index_type_string to convert enum value to string
+    r.append(";indextype=%s" % (cdt_ctx.index_type_string(aerospike.INDEX_TYPE_LIST)))
+    # use index_datatype_string to convert enum value to string
+    r.append(";indexdata=string_list,%s" % (cdt_ctx.index_datatype_string(aerospike.INDEX_STRING)))
+    r.append(";context=%s" % (bs_b4_cdt))
+    req = ''.join(r)
+
+    # print("req is ==========={}", req)
+    retobj = client.info_all(req, policy=None)
+    # print("res is ==========={}", res)
+    client.index_remove('test', 'test_string_list_cdt_index', policy)
+
     # Cleanup
     client.remove(key)
     client.close()
 '''
 import aerospike
 
-#'index_type' - must be one of 'default', 'list', 'mapkeys', 'mapvalues'
+"""
+Converts index_type enum value to string.
+
+Args:
+    index_type (int): The index_type to convert into equivalent string value.
+
+Returns:
+    (string) - must be one of 'default', 'list', 'mapkeys', 'mapvalues'
+
+"""
```

Comment:
https://peps.python.org/pep-0257/ docstrings should be the first statement of this function, so they come after the `def`.